compiler/testData/codegen/box/primitiveTypes/equalityWithObject/intEqualsNull.kt

```diff
@@ -0,0 +1,2 @@
+fun box(): String =
```

Comment:
Please also add bytecode test or convert this one to it: use as example compiler/testData/codegen/bytecodeText/conditions/ tests